src/main/io/vtx_string.c

```diff
@@ -43,23 +45,20 @@ const char * const vtx58BandNames[] = {
     "BOSCAM E",
     "FATSHARK",
     "RACEBAND",
+    "LOW RACE",
 };
 
-const char vtx58BandLetter[] = "-ABEFR";
+const char vtx58BandLetter[] = "-ABEFRL";
```

Comment:
(you can use `VTX_SA_BAND_COUNT`, it will skip trailing `'\0'`)

Also comment that `vtx58BandLetter[0]` is for overflow